webpack.config.dist.js

```diff
@@ -0,0 +1,23 @@
+require("babel/register");
```

Comment:
Here and in a couple other places in the same file, there's some inconsistency on quotes and spacing around parentheses. The pattern I'm seeing elsewhere in the code base is single quotes, no spaces between parentheses.